yarn.lock

```diff
@@ -1151,9 +1151,27 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@mui/utils@npm:^6.0.0-alpha.1":
```

Comment:
```suggestion
"@mui/utils@npm:^6.0.0-alpha.1":
```
This doesn't seem right...

Reply:
Looked into it a bit more - it seems this is from `@mui/base` which which are all labelled as beta versions anyway.